cli/cmd/inventory.go

```diff
@@ -0,0 +1,38 @@
+package cmd
```

Comment:
I don't know yet. Seems like this is the place where one would do things like searches, etc.? One could argue that they also will surface on the list commands? If you feel strongly, I can move it.